### PR TITLE
refactor: use spring Dependency Injection and refactor the route configuration

### PR DESCRIPTION
**Pull request by fdlk**

### CODE REVIEW

File: src/main/java/org/molgenis/MySpringBootRouter.java [hunk 1]
```diff
@@ -102,17 +114,17 @@ public void configure() {
     from("direct:map-alissa-result")
         .split().body()
         .process().body(Map.class, alissaTableMapper::mapLine)
-        .to(marshalVkglResults);
+        .to("direct:marshal-vkgl-result");
```
Comment (fdlk) [inline src/main/java/org/molgenis/MySpringBootRouter.java:118]: I know sonar isn't happy with repeated strings and in general I agree but for route names I think it actually is clearer than extracting constants.
Bit like in test cases, where repeating strings is also a good thing.

File: src/main/java/org/molgenis/MySpringBootRouter.java [hunk 2]
```diff
@@ -137,23 +170,11 @@ public void configure() {
     from("direct:hgvs2vcf")
         .description("Validates the normalized gDNA.")
         .aggregate(header(FILE_NAME), groupedBody())
-        .completionSize(DEFAULT_TIMEOUT)
+        .completionSize(COMPLETION_SIZE)
```
Comment (fdlk) [inline src/main/java/org/molgenis/MySpringBootRouter.java:173]: whoops :D

Comment (marikaris) [reply src/main/java/org/molgenis/MySpringBootRouter.java:173]: Haha whoops indeed, the number was equal though.

File: src/main/java/org/molgenis/MySpringBootRouter.java (hunk 1 above)
Comment (marikaris) [reply src/main/java/org/molgenis/MySpringBootRouter.java:118]: Agree. I just wanted to please the sonar gods. Should we silence them?

Comment (fdlk) [reply src/main/java/org/molgenis/MySpringBootRouter.java:118]: Make them go away! :)

**Pull request merged**